server/src/testFixtures/java/io/crate/lucene/CrateLuceneTestCase.java

```diff
@@ -159,6 +167,19 @@ public static TestRuleMarkFailure getSuiteFailureMarker() {
         }
     }
 
+    /**
+     * Ignores thread leaks from {@link ForkJoinPool#commonPool()} which is used by components like
+     * {@link HttpClient} and cannot be shutdown manually. It closes on SystemExit.
```

Comment:
Where do we config/or where its defaulted that `HttpClient` used `ForkJoinPool`?

Sometimes we use generic executor, for example in https://github.com/crate/crate/pull/16135/files#diff-82199e591de82391f38e46146255508554abd71c865d82f8f6b2d4efa29c27c6R136

and sometimes `newHttpClient()` which is afaik default `HttpClient` implementation using 

`ex = Executors.newCachedThreadPool(new DefaultThreadFactory(id));`

Reply:
Nevermind, https://stackoverflow.com/a/51982967.